starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,65 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+  <link rel="stylesheet" href="./css/main.css">
+  
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+<div class="whole-page">
+
+<section class = "title">
+    <h2>Apple Pie</h2> 
+</section>
+
+<br>
+<div class="page-content">
+<p>This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+
+<img class ="info" src="./images/recipe-info.png" alt="Recipe Info">
+
+<hr> 
+
+<div> 
+  <h3>Ingredients</h3>
+  <ul class="dashed">
+      <li> 1 recipe pastry for a 9 inch double crust pie</li>
+      <li> 1/2 cup unsalted butter</li>
+      <li> 3 tablespoons all-purpose flour</li>
+      <li> 1/4 cup water</li>
+      <li> 3 tablespoons all-purpose flour</li>
+      <li> 1/4 cup water</li>
+      <li> 1/2 cup white sugar</li>
+      <li> 1/2 cup packed brown sugar</li>
+      <li> 8 Granny Smith apples - peeled, cored and sliced</li>
+  </ul>
+
+<img class ="info" src="./images/cooking-info.png" alt="Cooking Info">
+
+</div>
+
+<hr>  
+
+<div> 
+<h3>Directions</h3>
+
+<ol>
+<li>Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
```

Comment:
this list should have had the number then ) instead of a period(.).  One way to do this is to just do `list-style: none` then add 1) 2) etc in front of the text of each list. or you can use the pro way and in your css folder do this 
`
ol {
counter-reset: custom;
list-style-type: none;
line-height: 1.6;
margin: 0 auto;
}

ol li:before {
content: counter(custom)') ';
counter-increment: custom;
}
`